spec/requests/carto/api/visualizations_controller_spec.rb

```
@@ -21,6 +21,10 @@
   include VisualizationDestructionHelper
   include FeatureFlagHelper
 
+  before(:all) do
+    create_account_type_fg('ORGANIZATION USER')
```

Same as above, Sequel 3 ignoring errors.